backend_kivy.py

```diff
@@ -1107,7 +1107,8 @@ def on_touch_down(self, touch):
            `motion_notify_event`, `scroll_event`, `button_press_event`,
            `enter_notify_event` and `leave_notify_event`
         '''
-        newcoord = self.to_widget(touch.x, touch.y, relative=True)
+        x0,y0 = self.to_window(touch.x,touch.y)
+        newcoord = self.to_widget(x0,y0, relative=True)
```

Comment:
You can do this more simply with `to_local`, instead of `to_window` followed by `to_widget`.

Reply:
Good point. Thank you for the simplification, I tested it and it works as intended.